contracts/HubPool.sol

```diff
@@ -26,6 +26,13 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 
+/**
+ * @notice Contract deployed on Ethereum that houses L1 token liquidity for all SpokePools. Admin can instruct this
```

Comment:
should we say "admin"? this is not really correct, no? it's more that a valid bundle does the action you describe.

Reply:
Yeah, this is definitely not an admin. Anyone can propose these and their validity is validated via the OO.

Reply:
sure